make build fails on Python 3.4 with latest pkg level due to Pygments version conflict

See here: https://travis-ci.org/github/pywbem/pywbemtools/jobs/662559173?utm_medium=notification&utm_source=github_status

Error message of make build surfacing the issue:
```
pkg_resources.VersionConflict: (Pygments 2.3.1 (/home/travis/virtualenv/python3.4.8/lib/python3.4/site-packages), Requirement.parse('Pygments>=2.5.1'))
Makefile:578: recipe for target 'dist/pywbemtools-0.5.1.dev57-py2.py3-none-any.whl' failed
```

Earlier error message during `make develop`:
```
ERROR: readme-renderer 25.0 has requirement Pygments>=2.5.1, but you'll have pygments 2.3.1 which is incompatible.
```

Dependencies:
* Collecting Pygments>=2.0 (from Sphinx>=1.7.6->-r dev-requirements.txt (line 37))
* Collecting readme-renderer>=21.0 (from twine>=1.8.1->-r dev-requirements.txt (line 53))

--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -38,6 +38,10 @@ Sphinx>=1.7.6
 sphinx-git>=10.1.1
 GitPython>=2.1.1;
 sphinxcontrib-fulltoc>=1.2.0
+# Pygments 2.4.0 has removed support for Python 3.4
+Pygments>=2.1.3; python_version == '2.7'
+Pygments>=2.1.3,<2.4.0; python_version == '3.4'
+Pygments>=2.1.3; python_version >= '3.5'
 
 # PyLint (no imports, invoked via pylint script)
 # Pylint requires astroid
@@ -51,6 +55,10 @@ flake8>=3.2.1;
 
 # Twine (no imports, invoked via twine script):
 twine>=1.8.1
+# readme-renderer 25.0 has removed support for Python 3.4
+readme-renderer>21.0; python_version == '2.7'
+readme-renderer>21.0,<25.0; python_version == '3.4'
+readme-renderer>21.0; python_version >= '3.5'
 
 # Jupyter Notebook (no imports, invoked via jupyter script)
 # TODO Future: currently jupyter not used by pywbemtools

--- a/docs/changes.rst
+++ b/docs/changes.rst
@@ -54,6 +54,9 @@ Released: not yet
 * Test: Fixed dependency to Python development packages on CygWin platform
   in Appveyor CI.
 
+* Pygments 2.4.0 and readme-renderer 25.0 have removed support for Python 3.4
+  and have therefore been pinned to below these versions on Python 3.4.
+
 **Enhancements:**
 
 * Promoted development status of pywbemtools from Alpha to Beta.

